### DISCUSSION TITLE
Is epicweb-dev/totp the right choice for implementing passwordless login?

### DISCUSSION THREAD

**Discussion by MoSattler** [epicweb-dev/epic-stack#856] [Q&A] [answered]
Hello everyone!

I’m looking to remove passwords entirely from my Epic Stack app and instead use “magic links” or codes sent to users’ emails for login. I believe that epicweb-dev/totp is well-suited for this task, but I wanted to check with the community to see if there are any security considerations I might be overlooking that could make epicweb-dev/totp a less ideal choice.

**Comment by kentcdodds** [comment, marked as answer]
The totp solution works great for that use case 👍
